src/endpoint/s3/ops/s3_put_bucket_lifecycle.js

```diff
@@ -9,6 +9,42 @@ const S3Error = require('../s3_errors').S3Error;
 
 const true_regex = /true/i;
 
+// validates lifecycle rule
+function validate_lifecycle_rule(rule) {
```

Comment:
we dont support `transitions` in Noobaa for now, can you add a validation for that. Maybe an error message if someone add a `transitions` to lifecycle.

Reply:
It is already supported. Thanks